common/utils.js

```diff
@@ -453,6 +453,17 @@ const getItsEdgeContract = (chainConfig) => {
     return itsEdgeContract;
 };
 
+const getItsEdgeChains = (config, excludeChainName) => {
```

Comment:
Self chain name shouldn't be excluded. There is no technical issue with sending an ITS transfer to the same chain, similar to sending a GMP call to the same chain, so self chain name should be whitelisted